dmutils/content_loader.py

```diff
@@ -26,11 +30,16 @@ def __init__(self, sections):
         for section in self.sections:
             for question in section.questions:
                 question_index += 1
-                question['number'] = question_index
+                question.number = question_index
 
     def __iter__(self):
         return self.sections.__iter__()
 
+    def summary(self, service_data):
```

Comment:
I think this could do with a docstring. Without reading the commit comment it wasn't immediately obvious what this was for.